src/widgets/unfold_widgets/preprocessing_widget.py

```diff
@@ -154,7 +154,7 @@ def _clear_plot(self):
 
     def add_columns_to_layout(self):
         self.clear_column_layout()
-        columns = self.engine.get_columns()
+        columns = self.engine.get_raw_columns()
```

Comment:
I really enjoy that solution, but here I am not a fan of the previous solution with clearing checkboxes after submitting. Maybe we can keep a checkbox state after clicking select actually? Talking about line 160.